projects/radare2/default.options

```diff
@@ -0,0 +1,4 @@
+[libfuzzer]
+close_fd_mask=3
+detect_leaks=0
+rss_limit_mb=0
```

Comment:
Not a great idea. The workers on oss-fuzz have small amount of RAM anyway,
but with this flag the OOM failures will become cryptic.

Reply:
The memory consumption is high, I will try to work on the fuzzer / radare2 to see if we can make it to consume less memory.

Reply:
I believe that we'll simply ignore value of 0 as well as any value bigger than our default of 2048